react/components/ProductResources.tsx

```diff
@@ -1,7 +1,188 @@
+                loading ? (
+                  <Spinner />
+                ) : (
+                  <PageHeader
+                    title={data ? data.product.name : 'Error fetching product'}
+                    linkLabel={'Resources Manager'}
+                    onLinkClick={() => {
+                      context.navigate({ page: HOME_PAGE })
+                    }}
+                  />
+                )
+              }>
+              <PageBlock>
+                <ToastProvider positioning="window">
+                  <Table
+                    loading={loading}
+                    fullWidth
+                    emptyStateLabel={
+                      error
+                        ? `An error occurring while fetching resources: ${error.message}`
+                        : 'No resources found'
+                    }
+                    schema={jsonSchema}
+                    items={filteredData.slice(itemFrom, itemTo)}
+                    onRowClick={({ rowData }: any) => {
+                      context.navigate({
+                        page: RESOURCE_EDITION_PAGE,
+                        params: { resourceId: rowData.id },
+                      })
+                    }}
+                    toolbar={{
+                      inputSearch: {
+                        value: searchValue,
+                        placeholder: 'Search by name or key',
+                        onChange: (e: any) => setSearchValue(e.target.value),
+                        onClear: (e: any) => {
+                          e.preventDefault()
+                          setFilteredData(data.product.resources)
+                        },
+                        onSubmit: (e: any) => {
+                          e.preventDefault()
+                          const searchString: string = searchValue.toLowerCase()
+                          const newData = data.product.resources.filter(
+                            (resource: Resource) => {
+                              const resourceName = resource.name.toLowerCase()
+                              const resourceKey = resource.resourceKey.toLowerCase()
+                              return (
+                                resourceName.includes(searchString) ||
+                                resourceKey.includes(searchString)
+                              )
+                            }
+                          )
+                          setFilteredData(newData)
+                        },
+                      },
+                      newLine: {
+                        label: 'New Resource',
+                        handleCallback: () =>
+                          context.navigate({
+                            page: RESOURCE_CREATION_PAGE,
+                            params: { productId: data.product.id },
+                          }),
+                      },
+                    }}
+                    pagination={{
+                      onNextClick: (e: any) => {
```

Comment:
In the `onNextClick` handler, as well as in the `onPrevClick` handler, it would be good to add a line that pull the screen "up". 
```suggestion
                      onNextClick: (e: any) => {
                      window.scrollTo(0, 0)
```